src/layouts/default.html.eco

```diff
@@ -64,7 +64,7 @@
 					<% end %>
 					<li>
 						<a href="//discourse.transformap.co">
-							Forum
+							Agora
```

Comment:
This needs to be changed to **Discourse** or **Community** (at will) in following the unrepresentative survey at http://discourse.transformap.co/t/domain-name-for-this-discourse-forum/383/11

Forum is just out, sorry.